uaclient/messages/__init__.py

```diff
@@ -272,6 +272,24 @@ class TxtColor:
 APT_REMOVING_SOURCE_FILE = "Removing apt source file: {filename}"
 APT_REMOVING_PREFERENCES_FILE = "Removing apt preferences file: {filename}"
 
+# Kernel checks for Purge
+PURGE_KERNEL_REMOVAL = (
+    "Purging the fips packages would uninstall the following kernel(s):"
```

Comment:
I think this message could happen for disabling realtime-kernel with --purge as well

Reply:
true will make the service a variable